src/components/TreeNode.tsx

```diff
@@ -118,20 +208,89 @@ const TreeNode = ({
     setY(_y)
   }, [_y])
 
+  useLayoutEffect(() => {
+    if (dragInProgress) return
+
+    const element = fadeThoughtRef.current
+    if (!element) return
+
+    if (!hasMounted.current) {
+      hasMounted.current = true
+      prevIsTableCol1.current = isTableCol1
+      return
+    }
+
+    // If we've already run once and the column flag didn't change, skip animation.
+    if (hasMounted.current && prevIsTableCol1.current === isTableCol1) {
+      return
+    }
+
+    // Mark that the effect has now run at least once.
+    hasMounted.current = true
+
+    // Grab the bullet and editable elements.
+    const bulletEl = element.querySelector('[aria-label="bullet"]') as HTMLElement
+    const editableEl = element.querySelector('.editable') as HTMLElement
```

Comment:
We also need to animate https://github.com/cybersemics/em/blob/203e63ba5c24deb4f32a9e729bfa68897e1451c7/src/components/ThoughtAnnotation.tsx since it contains the superscript and url icon.

Unless it's possible to animate a single container rather than all the elements individually?

```
- a
  - m
    - x
  - test.com
    - y
  - this is a longer thought
- b
  - m
    - y
```

https://github.com/user-attachments/assets/8a6e30c5-774a-4512-b89b-f33bfb3b65ef